pkgs/by-name/sq/sqlitestudio/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libsForQt5,
+  readline,
+  tcl,
+  python3,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+stdenv.mkDerivation rec {
+  pname = "sqlitestudio";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "pawelsalawa";
+    repo = "sqlitestudio";
+    rev = version;
+    hash = "sha256-5oBYv8WxyfVvvqr15XApvn6P/lBxR8b6E+2acRkvX0U=";
+  };
+
+  nativeBuildInputs =
+    with libsForQt5.qt5;
+    [
+      qmake
+      qttools
+      wrapQtAppsHook
+    ]
+    ++ [ copyDesktopItems ];
+
+  buildInputs =
+    with libsForQt5.qt5;
+    [
+      qtbase
+      qtsvg
+      qtdeclarative
+      qtscript
+    ]
+    ++ [
+      readline
+      tcl
+      python3
+    ];
+
+  qmakeFlags = [ "../../SQLiteStudio3" ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "SQLiteStudio";
+      exec = pname;
+      icon = pname;
+      comment = "Database manager for SQLite";
+      terminal = false;
+      startupNotify = false;
+      categories = [ "Development" ];
+    })
+  ];
+
+  preConfigure = ''
+    mkdir -p output/build
+    cd output/build
+  '';
```

Comment:
This should not be necessary?

Reply:
Yes indeed, this is not necessary and has been removed.
`qmakeFlags` and `postInstall` instructions has been modified accordingly. Thank you!